components/dialog/overview.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Overview
+page_title: Dialog Overview
+description: Overview of the Dialog for Blazor.
+slug: dialog-overview
+tags: telerik,blazor,dialog,overview
+published: True
+position: 0
+---
+
+# Dialog Overview
+
+The Dialog component is a modal popup that displays information to the user. It is fully customizable and extends the `DialogBuilder` to provide you with a way to achieve complex UI popups.
```

Comment:
* This introduction should highlight the core purpose and most valuable feature of the component. It should answer the question "Why / when should I use it?" What is the difference between the Dialog and the Window?
* "fully customizable" - marketing phrase, doesn't mean anything
* "Displays infomation to the user" is too generic - each of our components does that.
* "Extends the DialogBuilder" - this is an implementation detail that may not belong here, unless it's really important.
* "complex UI popups" - what is a complex popup?